### PR TITLE
Implement fake database generator [Oleksii]

### PR DESCRIPTION
**Pull request by vglfr**

### CODE REVIEW

File: server/src/models/fake.database.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        tags: words(number(6)).split(' '),
+    }));
+
+    return await Post.insertMany(posts);
+};
+
+const generateLikes = async function(users: IUserModel[], posts: IPostModel[], size: number): Promise<ILikeModel[]> {
+    const { random: { arrayElement } }: any = faker; // FIXME any type
+
+    const likes = Array(size).fill(null).map((throwaway: null) => ({
```
Comment (InnaKutsik) [inline server/src/models/fake.database.ts]: new Array(size....

File: server/src/models/fake.database.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    return await Token.insertMany(tokens);
+};
+
+const generatePosts = async function(users: IUserModel[], size: number): Promise<IPostModel[]> {
+    const {
+        random: { arrayElement, alphaNumeric, number },
+        lorem: { words },
+    }: any = faker; // FIXME any type
+
+    const posts = Array(size).fill(null).map((throwaway: null) => ({
```
Comment (InnaKutsik) [inline server/src/models/fake.database.ts]: new Array(size....

File: server/src/models/fake.database.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        isVerified: true,
+    }));
+
+    return await User.insertMany(users);
+};
+
+const generateTokens = async function(users: IUserModel[], size: number): Promise<ITokenModel[]> {
+    const { random: { arrayElement, alphaNumeric } }: any = faker; // FIXME any type
+
+    const tokens = Array(size).fill(null).map((throwaway: null) => ({
```
Comment (InnaKutsik) [inline server/src/models/fake.database.ts]: new Array(size....

File: server/src/models/fake.database.ts (hunk 1 above)
Comment (vglfr) [reply server/src/models/fake.database.ts]: fixed

File: server/src/models/fake.database.ts (hunk 2 above)
Comment (vglfr) [reply server/src/models/fake.database.ts]: fixed

File: server/src/models/fake.database.ts (hunk 3 above)
Comment (vglfr) [reply server/src/models/fake.database.ts]: fixed

**Pull request merged**